unittests/errors/pan/functions/host_domain_from_object3.pan

```diff
@@ -0,0 +1,6 @@
+object template host_domain_from_object3;
+
+include 'pan/functions';
+
+@{fails with invalid shorthostname due to _ in name}
```

Comment:
the error is thrown by merge in that case, so i will also allow `evaluation` errors